android/java/org/chromium/chrome/browser/rewards/BraveRewardsPanel.java

```diff
@@ -759,27 +770,41 @@ public void showLikePopDownMenu() {
         requestPublisherInfo();
         fetchRewardsData();
         setNotificationsControls();
+        String rewardsCountryCode = UserPrefs.get(Profile.getLastUsedRegularProfile())
+                                            .getString(BravePref.DECLARED_GEO);
         if (mPopupView != null && PackageUtils.isFirstInstall(mActivity)
```

Comment:
Curious: do we still need these `mPopupView != null` checks here?

Reply:
It's safe to check for mPopupview as we open rewards panel from several places in android.

Reply:
Sure, I was just thinking that, given that now we don't use it here directly, maybe the null-check would be more appropriate where we actually use it. But I'll leave that up to you. 🙂